app/src/main/java/com/radu/newsreader/model/article/ArticleItemViewModel.java

```diff
@@ -0,0 +1,19 @@
+package com.radu.newsreader.model.article;
+
+import androidx.lifecycle.LifecycleObserver;
+import androidx.lifecycle.ViewModel;
+
+public class ArticleItemViewModel extends ViewModel implements LifecycleObserver {
+    // TODO: Implement the ViewModel
```

Comment:
let's clean this TODO and make fields final, a final field means we cannot change the state for this class after initial creation, less states means less bugs.